app/routes/datasets/dataset.py

```diff
@@ -25,6 +25,16 @@
 router = APIRouter()
 
 
+async def get_owner(
+    dataset: str = Depends(dataset_dependency), user: User = Depends(get_manager)
+) -> User:
+    dataset_row: ORMDataset = await datasets.get_dataset(dataset)
```

Comment:
Can you add a """ .... """ comment line describing the function above this line?

Reply:
Will do, thanks! Also, I'm adding more tests.